contents/docs/posthog-ai/pricing-faq.mdx

```diff
@@ -0,0 +1,61 @@
+# PostHog AI credits
+
+PostHog uses **AI credits** to charge for all of its AI features.  
+Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — credits are used based on the effort required to complete your request.
+
+This means you only pay for what you actually use.  
+There’s no fixed subscription — and every organization has a free tier worth $25 (2,500 credits) so you can explore all PostHog’s AI features before paying anything.
+
+### When AI credits are used
+
+AI credits are consumed whenever PostHog performs intelligent work for you.  
+You’ll spot these features by the ✨ icon or when using the in-app chat.
```

Comment:
```suggestion
AI credits are consumed whenever PostHog performs intelligent work for you.  You’ll spot these features by the ✨ icon or when using the in-app chat.
```

@edwinyjlim maybe we rebase this on your [other PR](https://github.com/PostHog/posthog.com/pull/13614#pullrequestreview-3448256894) and link to the list of actions here (if we have a list)